koku/koku/middleware.py

```diff
@@ -105,33 +107,34 @@ def __init__(self, dikt):
         super().__init__(data)
 
 
-class KokuTenantSchemaExistsMiddleware(MiddlewareMixin):
-    """A middleware to check if schema exists for Tenant."""
+# TODO: Figure out if we still need this?
+# class KokuTenantSchemaExistsMiddleware(MiddlewareMixin):
```

Comment:
Chatted with the team and we are going to remove this piece of middleware. Dan will be able to get the same information from the source api with the new settings UI moving forward.